tests/bench_msg_pingpong/main.c

```diff
@@ -29,22 +29,20 @@
 #define TEST_DURATION       (1000000U)
 #endif
 
-volatile unsigned _flag = 0;
 static char _stack[THREAD_STACKSIZE_MAIN];
 
 static void _timer_callback(void*arg)
 {
-    (void)arg;
-
-    _flag = 1;
+    unsigned *flag = arg;
+    *flag = 1;
 }
 
 static void *_second_thread(void *arg)
 {
     (void)arg;
-    msg_t test;
 
     while(1) {
```

Comment:
```suggestion
    while (1) {
```

Care to also fix this style issue?